### ISSUE TITLE
Example Box CSS

### ISSUE THREAD

**Issue by johnRedden** [johnRedden/TextbookProject#11]
Need to find the easiest way to wrap each example in a div so that they can be boxed nicely.

If we can incorporate the solution into the ckEditor that would be great.

**Comment by FelixVicis** [comment]
Sorry, I need a bit more info. Each example of what?

**Comment by johnRedden** [comment]
Just need special styling for examples in the book.  Something like this:

```
        .ens-example {
            background: #eee;
            border: 1px solid #ccc;
            padding: 5px 10px;
        }

```

**Comment by FelixVicis** [comment]
Couldn't we make a special button for that with ckEditor then? I could've sworn it was in the documentation.

**Comment by johnRedden** [comment]
Yes, I think so. OR we could click the little div button and add the class "ens-example" ... something.  This is low priority.

**Comment by FelixVicis** [comment]
I just updated #10 with a question. While in between topics, I'll go see if there is a quick fix in the documentation.

**Comment by FelixVicis** [comment]
Okay, found an example. This is something that would have to be added after we've downloaded the ckEditor source. 
- Example from the ckEditor [How to make a plugin](http://docs.ckeditor.com/#!/guide/plugin_sdk_sample) page.

**Comment by johnRedden** [comment]
Also, we need to learn how to eliminate buttons.  I think the form, checkbox, radio button group needs to go.

**Comment by johnRedden** [comment]
Eliminated the buttons, added the save button.  Only need to figure out how to add the custom example box css to the workflow.

**Comment by FelixVicis** [comment]
I believe this to be closed as of 7fbb149abfb82aac808a2d441184d36ece7e01cd, closing in @johnRedden 's stead.
